Organization/Organization/CreateEmployeeViewController.h

```diff
@@ -1,11 +1,8 @@
 #import <UIKit/UIKit.h>
 #import "SavedProtocol.h"
 
-
-
-
 @interface CreateEmployeeViewController : UIViewController
 
-@property (nonatomic, readwrite) id <SaveEmployeeProtocol> saver;
+@property (nonatomic) id <SaveEmployeeProtocol> delegate;
```

Comment:
Each property should have memory management type attribute. In this case it's `assign`, which shouldn't be applied for objects.